doc/nrf/releases/release-notes-latest.rst

```diff
@@ -222,24 +227,29 @@ The following list summarizes the most important changes inherited from upstream
 * Added support for invoking shell commands (shell management) from the mcumgr command line.
 
 
+Multiprotocol Service Layer (MPSL)
```

Comment:
MPSL is part of sdk-nrfxlib, so this section should be moved up (line 172/177) and made a subsection (~~~)